### DISCUSSION TITLE
MAUI app crashes on Android Device with Segfault error

### DISCUSSION THREAD

**Discussion by imsam67** [dotnet/maui#8794] [Q&A]
My .NET MAUI app runs fine on Android Emulator and a tethered physical device. However, once I package it and upload it to Google Play for internal testing, it instantly crashes on the very same device I was using for testing i.e. the tethered device.

I use Sentry for crash logs and all I see in the crash logs is the following:

SIGSEGV
Segfault

split_config.armeabi_v7a.apk

Any idea what this is and how to resolve it?

**Comment by xtuzy** [comment]
Do you try install app not from VisualStudio? Are you use Release mode?

**Comment by kentvdc029** [comment]
I am also experiencing this. Did you find the solution?

**Comment by imsam67** [comment]
It was an error in my code. I don't remember the details but the error was definitely caused by my own code.
